app/services/catalog/create_approval_request.rb

```diff
@@ -44,7 +44,13 @@ def request_body_from(order_item)
     end
 
     def workflows(portfolio_item)
-      portfolio_item.resolved_workflow_refs
+      workflows = portfolio_item.resolved_workflow_refs
+      case workflows
+      when :empty?.to_proc
+        raise Catalog::ApprovalError, "Portfolio Item has no Approval workflows to run"
```

Comment:
Possibly: No Approval workflows assigned
The current text seems to suggest there were no approval workflows available which should never be true.

Reply:
Updated :+1: